File: Nível 2/1043 - Triângulo/main.lua
valores = io.read()
x1, x2, x3 = valores:match("(%S+)%s+(%S+)%s+(%S+)")
a = tonumber(x1)
b = tonumber(x2)
c = tonumber(x3)

perimetro = a + b + c

if math.abs(b - c) < a and a < (b + c) and math.abs(a - c) < b and b < (a + c) and math.abs(a - b) < c and c < (a + b) then
    print(string.format("Perimetro = %.1f", a + b + c))
else
    area = ((a + b) / 2) * c 
    print(string.format("Area = %.1f", area))
end

File: Nível 2/1043 - Triângulo/main.py
valores = input("A ")

x1,x2,x3 = valores.split()

a = float(x1)
b = float(x2)
c = float(x3)

if abs(b - c) < a < (b + c) and (a - c) < b < (a + c) and (a - b) < c < (a + b):
    print(f'Perimetro = {a+b+c:.1f}')
else:
    area = ((a + b) / 2) * c
    print(f'Area = {area:.1f}')